flox-bash/lib/bootstrap.sh

```diff
@@ -1,3 +1,4 @@
+# -*- mode: sh; sh-shell: bash; -*-
```

Comment:
I’m unfamiliar with what this line is doing

Reply:
It tells emacs to treat the file as bash code so it runs the correct LSP and linter